WORKSPACE

```diff
@@ -40,6 +40,22 @@ http_archive(
     urls = ["https://github.com/kubernetes/repo-infra/archive/df02ded38f9506e5bbcbf21702034b4fef815f2f.tar.gz"],
 )
 
+skylib_version = "0.8.0"
```

Comment:
The variable here feels a bit misleading since updating it would also require updating the sha256 in the http_archive definition.

Reply:
that's a fair point, I copy/pasta'd and should have used more discretion. I will merge this version into the URL like we do in the above archives